sdk/core/Azure.Core/samples/README.md

```diff
@@ -16,5 +16,5 @@ description: Samples for the Azure.Core client library
 - [Long Running Operations](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md)
 - [Events](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Events.md)
 - [Diagnostics](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md)
-- [Mocking](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Mocking.md)
+- [Mocking](https://learn.microsoft.com/dotnet/azure/sdk/unit-testing-mocking)
```

Comment:
Should we consider making this an aka.ms link so that we can control it and get usage data?

Reply:
Since the doc is hosted on Learn, we already get great telemetry via an internal dashboard. We also can easily implement redirects if this document path were to change. With that being said, do you still want an aka.ms link?

Reply:
no, sounds like we are covered.